ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-release-4.20__periodics.yaml

```diff
@@ -0,0 +1,515 @@
+base_images:
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  ocp_4.20_base-rhel9:
+    name: "4.20"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-9-golang-1.24-openshift-4.20:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.24-openshift-4.20
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.rhel7
+  inputs:
+    ocp_4.20_base-rhel9:
+      as:
+      - registry.ci.openshift.org/ocp/4.20:base-rhel9
+    ocp_builder_rhel-9-golang-1.24-openshift-4.20:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20
+  to: cluster-authentication-operator
+promotion:
+  to:
+  - disabled: true
+    name: "4.20"
+    namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.20"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.20"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-aws-external-oidc-configure
+  interval: 24h
+  steps:
+    cluster_profile: aws-3
+    env:
+      OPENSHIFT_SKIP_EXTERNAL_TESTS: "True"
+      TEST_SKIPS: (\[OCPFeatureGate:ExternalOIDCWithUIDAndExtraClaimMappings\])|(\[OCPFeatureGate:ExternalOIDC\]
+        reverting to IntegratedOAuth)
+      TEST_SUITE: openshift/auth/external-oidc
+    workflow: openshift-e2e-aws
+- as: e2e-azure-external-oidc-configure
+  interval: 24h
+  steps:
+    cluster_profile: azure4
+    env:
+      OPENSHIFT_SKIP_EXTERNAL_TESTS: "True"
+      TEST_SKIPS: (\[OCPFeatureGate:ExternalOIDCWithUIDAndExtraClaimMappings\])|(\[OCPFeatureGate:ExternalOIDC\]
+        reverting to IntegratedOAuth)
+      TEST_SUITE: openshift/auth/external-oidc
+    workflow: openshift-e2e-azure
+- as: e2e-gcp-external-oidc-configure
+  interval: 24h
+  steps:
+    cluster_profile: gcp
+    env:
+      OPENSHIFT_SKIP_EXTERNAL_TESTS: "True"
+      TEST_SKIPS: (\[OCPFeatureGate:ExternalOIDCWithUIDAndExtraClaimMappings\])|(\[OCPFeatureGate:ExternalOIDC\]
+        reverting to IntegratedOAuth)
+      TEST_SUITE: openshift/auth/external-oidc
+    workflow: openshift-e2e-gcp
+- as: e2e-vsphere-external-oidc-configure
+  interval: 24h
```

Comment:
Is it possible to choose some jobs to cover other ARCH like multi-arch or amd64? I intended to cover it to be tracked in https://issues.redhat.com/browse/AUTH-574 . Happened to see you're adding the vsphere platform jobs.

Reply:
We would have to add those in addition to the ones configured here.

I'm certainly open to it, but this PR is specifically scoped to adding the jobs required to satisfy the OCP feature promotion criteria which does not include `arm64` or `multi-arch` jobs.

I'll do some further exploration and if it is straightforward to add arm64 and/or multi-arch jobs I'll add them. Otherwise, I'll suggest that we add those in a follow-up PR so we can continue making progress towards feature promotion.